game/scripts/vscripts/Sections/Chapter4/SectionOutpost.ts

```diff
@@ -37,40 +43,48 @@ function onStart(complete: () => void) {
     playerHero.SetAttackCapability(UnitAttackCapability.MELEE_ATTACK);
 
     const direOutpost = getOrError(Entities.FindByName(undefined, "npc_dota_watch_tower_top"));
+    allowUseItem = false;
 
     graph = tg.withGoals(_ => goalTracker.getGoals(),
         tg.seq([
-            tg.setCameraTarget(playerHero),
-            tg.wait(1),
-
+            tg.immediate(_ => shared.blockades.direJungleLowToHighground.destroy()),
```

Comment:
A recurring thing in this chapter that I see is the instant destruction of the low to highground barrier. Spawning barriers is somewhat performance heavy, do we need to do that every section? 

Maybe it would be better to not create that barrier - put it in another array and only call the array of needed barriers?

Reply:
Agree with your suggestion, yet no critical performance issue found so far, I think it's okay for now.

Reply:
Make an issue to fix that later so we won't forget please.

Reply:
#233  Issue created, conversation resolved.